packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1544,6 +1545,69 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll Custom HOTP Factor
+
+Enrolls a user for a Custom HOTP factor. The enrollment process involves passing a factor profile Id and shared secret for a particular token.  Note: Currently only auto-activation is supported for Custom HOTP Factor. 
```

Comment:
We should expand HOTP during its first use here.

Notes should use our note format:

```
> Note: Text goes here.
```